floodwatch/src/js/components/ProfileOptions.js

```diff
@@ -0,0 +1,262 @@
+    super(props);
+    this.state = setInitialStateLocation();
+  }
+
+  componentWillReceiveProps(nextProps: LocationPropsType) {
+    if (nextProps.userData) {
+      this.decodeTwoFishes(nextProps.userData.twofishes_id)    
+    }
+  }
+
+  async updateList(value: string) {
+    const val = await FWApiClient.get().getLocationOptions(value);
+    this.setState({items: val.interpretations, loading: false});
+  }
+
+  async decodeTwoFishes(id: string) {
+    const place = await FWApiClient.get().getDecodedLocation(id);
+    this.setState({
+      value: place.interpretations[0].feature.displayName
+    })
+
+  }
+
+  toggleDescriptionVisibility() {
+    const curVisibility = this.state.isDescriptionOpen;
+    this.setState({
+      isDescriptionOpen: !curVisibility
+    })
+  }
+
+  render() {
+    return (
+      <Row className="demographic-category">
+        <Col xs={12}>
+          <h4>{this.props.filter.question}</h4>
+          <div className="input-wrapper">
+          <Autocomplete
+            menuStyle={{zIndex: 1000}}
+            inputProps={{name:'country', id: 'location-autocomplete'}}
+            value={this.state.value}
+            items={this.state.items}
+            getItemValue={(item) => item.feature.displayName}
+
+            onChange={(event, value) => {
+              this.setState({ value, loading:true})
+              this.updateList(value);
+            }}
+
+            onSelect={(value, item) => {
+              this.setState({value: value, items: [item]})
+              this.props.updateLocation(item.feature.longId)
+            }}
+
+            renderItem={(item, isHighlighted) => (
+              <div style={(isHighlighted) ? this.state.highlightedStyle : this.state.regularStyle}>
+              {item.feature.displayName}
+              </div>
+            )}
+          />
+          </div>
+          <br/>
+          <Button className="learn-more" bsSize="xsmall" onClick={this.toggleDescriptionVisibility.bind(this)}>Learn more</Button>
+          <p style={{
+            display: (this.state.isDescriptionOpen)? 'block' : 'none'
+          }}>
+            <Well bsSize="small">{this.props.filter.why}</Well>
+          </p>
+        </Col>
+      </Row>
+    )
+  }
+}
+
+// default ------------------------------------------------------------------------------------------------------------------------
+
+type DefaultStateType = {
+  isDescriptionOpen: boolean
+};
+
+function getInitialStateDefault() {
+  return {
+    isDescriptionOpen: false
+  }
+}
+
+export class DefaultOption extends Component {
+  state: DefaultStateType
+
+  constructor() {
+    super();
+    this.state = getInitialStateDefault();
+  }
+
+  toggleDescriptionVisibility() {
+    const curVisibility = this.state.isDescriptionOpen;
+    this.setState({
+      isDescriptionOpen: !curVisibility
+    })
+  }
+
+  render() {
+    let elems;
+    if (this.props.userData) {
+      if (this.props.filter.name == 'age') {
```

Comment:
I don't think you need any of these if / else ifs since everything will be neither age nor country